components/ArticleReply/ReplyShare.js

```diff
@@ -0,0 +1,123 @@
+import { useState, useCallback } from 'react';
+import {
+  Popper,
+  Fade,
+  Paper,
+  ClickAwayListener,
+  Snackbar,
+} from '@material-ui/core';
+import { t } from 'ttag';
+import { makeStyles } from '@material-ui/core/styles';
+import CopyButton from './CopyButton';
+import cx from 'clsx';
+
+const useStyles = makeStyles(theme => ({
+  mobile: {
+    display: 'block',
+    [theme.breakpoints.up('md')]: { display: 'none' },
+  },
+  desktop: {
+    display: 'none',
+    [theme.breakpoints.up('md')]: { display: 'block' },
+  },
+  button: ({ open }) => ({
+    borderRadius: 45,
+    padding: '1px 8px',
+    outline: 'none',
+    cursor: 'pointer',
+    marginLeft: 12,
+    border: `1px solid ${
+      open ? theme.palette.primary[500] : theme.palette.secondary[100]
+    }`,
+    color: open ? theme.palette.primary[500] : theme.palette.secondary[200],
+    background: theme.palette.common.white,
+    [theme.breakpoints.up('md')]: {
+      padding: '4px 18px',
+      marginRight: 10,
+    },
+    '&:hover': {
+      border: `1px solid ${theme.palette.secondary[300]}`,
+      color: theme.palette.secondary[300],
+    },
+  }),
+  menu: {
+    marginTop: 40,
+  },
+}));
+
+const SUCCESS = 'SUCCESS';
+
+const ReplyShare = ({ copyText }) => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [message, setMessage] = useState('');
+  const [status, setStatus] = useState(null);
+
+  const classes = useStyles({ open: !!anchorEl });
+
+  const openMenu = event => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const closeMenu = () => {
+    setAnchorEl(null);
+  };
+
+  const onSuccess = text => {
+    setMessage(text);
+    setTimeout(() => setStatus('SUCCESS'), 0);
+    closeMenu();
+  };
+
+  const onCopied = useCallback(() => onSuccess('Copied to clipboard!'));
+
+  const handleShare = () => {
+    if (window.navigator && window.navigator.share) {
+      navigator
+        .share({ text: copyText })
+        .then(() => onSuccess(t`Successfylly Shared!`))
+        .catch(() => {});
+    }
+  };
+
+  return (
+    <>
+      <button
+        type="button"
+        className={cx(classes.button, classes.desktop)}
+        onClick={openMenu}
+      >
+        {t`Share`}
+      </button>
+      <button
+        type="button"
+        className={cx(classes.button, classes.mobile)}
+        onClick={handleShare}
```

Comment:
On desktop browser with a narrow screen, we get an inactive button:

![share-no-use](https://user-images.githubusercontent.com/108608/82830907-9013b280-9ee9-11ea-89ac-1b8fb1744319.gif)

Root cause is as described above -- we only need one button that invokes `navigator.share` if it's available and shows dropdown otherwise.